domain/src/test/java/org/oppia/android/domain/translation/TranslationControllerTest.kt

```diff
@@ -421,66 +475,6 @@ class TranslationControllerTest {
     assertThat(selection.selectionTypeCase).isEqualTo(SELECTED_APP_LANGUAGE)
     assertThat(selection.selectedLanguage).isEqualTo(ENGLISH)
   }
-
-  @Test
-  fun testUpdateAppLanguage_uninitializedToSystem_returnsUninitializedSelection() {
```

Comment:
For these removed tests, shouldn't these still pass in the new implementation? Why is it correct to remove them?

Reply:
updated.

Reply:
Ben i have updated the tests correctly to pass with usage of PersistenceCacheStore, maybe i do not fully understand the coverage of tests. Please let me know if this is not correct. Having an issue as implementation currently the returned value are permanently not in default instance. Failing to understand if this should be the expected behavior when using PersistenceCacheStore.

Reply:
This might've been a quirk with a previous review view that I had--it seems more like the tests are being updated, now, than removed.